include/acl_hal_mmd.h

```diff
@@ -10,7 +10,112 @@
 
 //////////////////////////////////////////////////////////////////////////
 //                                                                      //
-// Implementation of HAL that builds on top of aocl_mmd (board vendor    //
+// Versioning constants, address maps, and bit/byte positionings used   //
+// in acl_kernel_if and acl_pll                                         //
+//                                                                      //
+//////////////////////////////////////////////////////////////////////////
+
+//// acl_kernel_if
+// Versioning: This value must be read from addr 0
+// For unit tests to work, this defines must match the one in the unit test
+// header file
+#define KERNEL_VERSION_ID (0xa0c00001)
```

Comment:
It's great that those Macros are moved to a header file. I checked the original check-in of those values and couldn't see comments why they were put in the acl_kernel_if.c in the first place. I think the only thing needs to be cautious is to make sure those values are not defined duplicate in the compiler side as this header is included by the simulator.

Reply:
Yes I have removed the definition of these macros in the simulation library :)